examples/tutorials/california_house_price.py

```diff
@@ -0,0 +1,197 @@
+parser.add_argument("--epochs", type=int, default=100)
+parser.add_argument("--seed", type=int, default=0)
+parser.add_argument("--model", type=str,
+                    default="sentence-transformers/all-distilroberta-v1")
+parser.add_argument("--pooling", type=str, default="mean",
+                    choices=["mean", "cls"])
+parser.add_argument("--compile", action="store_true")
+args = parser.parse_args()
+
+torch.manual_seed(args.seed)
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+class SimpleTextToEmbedding:
+    def __init__(self, model: str, device: torch.device):
+        self.tokenizer = AutoTokenizer.from_pretrained(model)
+        self.model = AutoModel.from_pretrained(model).to(device)
+        self.device = device
+
+    def __call__(self, sentences) -> Tensor:
+        inputs = self.tokenizer(
+            sentences,
+            truncation=True,
+            padding="max_length",
+            return_tensors="pt",
+        )
+        for key in inputs:
+            if isinstance(inputs[key], Tensor):
+                inputs[key] = inputs[key].to(self.device)
+        out = self.model(**inputs)
+        return out.last_hidden_state[:, 0, :].detach().cpu()
+
+
+text_encoder = SimpleTextToEmbedding(model=args.model, device=device)
+
+kwargs = {
+        "text_stype": stype.text_embedded,
+        "col_to_text_embedder_cfg": TextEmbedderConfig(
+            text_embedder=text_encoder,
+            batch_size=5,
+        ),
+    }
+
+# Prepare datasets
+path = osp.join(osp.dirname(osp.realpath(__file__)), "..", "..", "data",
+                args.dataset)
+
+# Load Dataset
+dataset = MultimodalTextBenchmark(root=path, name=args.dataset, num_rows=5000, **kwargs)
+
+model_name = args.model.replace('/', '')
+filename = f"{model_name}_data.pt"
+dataset.materialize(path=osp.join(path, filename))
+dataset.tensor_frame.col_names_dict[stype.embedding] = dataset.tensor_frame.col_names_dict.pop(stype.text_embedded)
+dataset.tensor_frame.feat_dict[stype.embedding] = dataset.tensor_frame.feat_dict.pop(stype.text_embedded)
+
+is_classification = dataset.task_type.is_classification
+
+train_dataset, val_dataset, test_dataset = dataset[:0.8], dataset[0.8:0.9], dataset[0.9:]
+# Set up data loaders
+train_tensor_frame = train_dataset.tensor_frame
+
+val_tensor_frame = val_dataset.tensor_frame
+test_tensor_frame = test_dataset.tensor_frame
+train_loader = DataLoader(train_tensor_frame, batch_size=args.batch_size,
+                          shuffle=True)
+val_loader = DataLoader(val_tensor_frame, batch_size=args.batch_size)
+test_loader = DataLoader(test_tensor_frame, batch_size=args.batch_size)
+
+stype_encoder_dict = {
+    stype.categorical: EmbeddingEncoder(),
+    stype.numerical: LinearEncoder(),
+    stype.embedding: LinearEmbeddingEncoder(),
+    stype.multicategorical: MultiCategoricalEmbeddingEncoder(),
+    stype.timestamp: TimestampEncoder(na_strategy=NAStrategy.MEDIAN_TIMESTAMP)
+}
```

Comment:
The biggest message in the tutorial should be this. We want to convey the message that this is all we need to handle more complex columns. Other parts are similar to https://pytorch-frame.readthedocs.io/en/latest/get_started/introduction.html so we can omit them in the tutorial.